src/Hng.Web/Controllers/ProductController.cs

```diff
@@ -34,13 +44,13 @@ public async Task<ActionResult<ProductDto>> GetProductById(Guid id)
 
             var query = new GetProductByIdQuery(id);
             var product = await _mediator.Send(query);
-
             return Ok(new
```

Comment:
Theres a generic response dto in the shared foler in hng.application. return tyhat dto with your product dto set to the data field